app/controllers/common/MembersSummaryController.scala

```diff
@@ -72,16 +74,24 @@ class MembersSummaryController @Inject()(
       }
     }
 
+  private def getSelectedTaxYear(userAnswers: UserAnswers)(implicit messages: Messages): String = {
```

Comment:
I don't think messages is used. Also I'd consider putting this method into the TaxYear object rather than here

Reply:
Have addressed in https://github.com/hmrc/pension-scheme-event-reporting-frontend/pull/99/commits/14fbf3f505db6d7ddf31b9ae72c77f33cf34e127